app/components/QrCodeScanner/QrCodeScanner.jsx

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React, { Component } from 'react'
+
+import Button from '../Button'
+import Instascan from 'instascan'
+
+import qrCodeScannerStyles from './QrCodeScanner.scss'
```

Comment:
Could you use `import styles from ...` (to be consistent with other components)

Reply:
updated